empiric-ui/public/sitemap-0.xml

```diff
@@ -0,0 +1,28 @@
+<url><loc>https://empiric.network/details/eth-usd</loc><lastmod>2022-08-02T22:07:40.496Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://empiric.network/details/btc-usd</loc><lastmod>2022-08-02T22:07:40.496Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://empiric.network/details/sol-usd</loc><lastmod>2022-08-02T22:07:40.496Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://empiric.network/details/btc-eur</loc><lastmod>2022-08-02T22:07:40.496Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://empiric.network/details/avax-usd</loc><lastmod>2022-08-02T22:07:40.496Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://empiric.network/details/doge-usd</loc><lastmod>2022-08-02T22:07:40.496Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://empiric.network/details/shib-usd</loc><lastmod>2022-08-02T22:07:40.496Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://empiric.network/details/dai-usd</loc><lastmod>2022-08-02T22:07:40.496Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://empiric.network/details/usdt-usd</loc><lastmod>2022-08-02T22:07:40.496Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://empiric.network/details/usdc-usd</loc><lastmod>2022-08-02T22:07:40.496Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://empiric.network/details/tusd-usd</loc><lastmod>2022-08-02T22:07:40.496Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://empiric.network/details/busd-usd</loc><lastmod>2022-08-02T22:07:40.496Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://empiric.network/details/bnb-usd</loc><lastmod>2022-08-02T22:07:40.496Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://empiric.network/details/ada-usd</loc><lastmod>2022-08-02T22:07:40.496Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://empiric.network/details/xrp-usd</loc><lastmod>2022-08-02T22:07:40.496Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://empiric.network/details/matic-usd</loc><lastmod>2022-08-02T22:07:40.496Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://empiric.network/details/btc-usd-20220930</loc><lastmod>2022-08-02T22:07:40.496Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://empiric.network/details/btc-usd-20221230</loc><lastmod>2022-08-02T22:07:40.496Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
```

Comment:
feels like we should remove the assets because we'd rather focus on giving a lot of priority to the main pages? And the sitemap file becomes much easier to grep for any humans/engineers that have to look at it

Reply:
Hmm I guess that kind of makes sense. I'm not sure how priority works. We can then do...
1. keep `next-sitemap` and define a rule that excludes `/details/*`
2. create the sitemap manually since there are only 5 pages or so.

Reply:
There is also a 3rd option where we give `details/*` a very low priority (like 0.2).

Reply:
I like option 1 the best, because it will adapt as we add pages.

Reply:
Went with option 1. Additionally, changed the priority of the homepage and removed the `sitemap-index` for now.